fetch.bs

```diff
@@ -1408,18 +1469,50 @@ for each associated <a for="fetch group">fetch record</a> whose
 <a>connection</a> is identified by an <b>origin</b> (an
 <a for=/>origin</a>) and <b>credentials</b> (a boolean).
 
+<h4 id=connection-reuse>HTTP/2 Connection Reuse</h4>
```

Comment:
tokbind use with http/2 is defined in [draft-ietf-tokbind-https](https://tools.ietf.org/html/draft-ietf-tokbind-https) and [draft-ietf-tokbind-protocol](https://tools.ietf.org/html/draft-ietf-tokbind-protocol), so we should not do anything here that conflicts with the latter specs.  

WRT [draft-ietf-tokbind-https](https://tools.ietf.org/html/draft-ietf-tokbind-https), it seems we overlooked "http/2 connection reuse", so having it here in fetch is nominally OK, but since -tokbind-https is not completed yet, h2  connection reuse may get (appropriately, imv) added to the latter spec in the near term as a part of last call.